.github/workflows/BuildPR.yml

```diff
@@ -15,10 +15,13 @@ jobs:
         xcode: [ Xcode_14.3, Xcode_15.2.0 ]
         os: [ macos-13 ]
         architecture: [ x86_64 ]
-#        include:
-#          - xcode: Xcode
-#            os: ARM64
-#            architecture: arm64
+        include:
+          - xcode: Xcode_15.4
+            os: macos-14
+            architecture: arm64
+          - xcode: Xcode_16
+            os: macos-14
+            architecture: arm64
```

Comment:
Sorry for asking: do these `include` declarations supersede the 3 lines above that have an array of xcode, os, and architecture? If so, can these 3 lines above be removed?

Reply:
No it adds to x86 matrix

Reply:
A normal matrix is a multiplication, so you can't simple add arm64. This must be added to matrix separated